custom_components/google_photos/coordinator.py

```diff
@@ -193,7 +193,6 @@ async def set_current_media_with_id(self, media_id: str | None):
             )
             self.current_media_secondary = None
             self.current_media_cache = {}
-            self.async_update_listeners()
```

Comment:
Could you elaborate why this was needed?

Reply:
You are right this isn't needed